frappe/public/css/report-rtl.css

```diff
@@ -1,3 +1,6 @@
 .grid-report {
   direction: ltr;
 }
+.page-form .awesomplete > ul {
+    min-width: 100%;
```

Comment:
`left: auto;` is a better solution, I think.

Reply:
Thanks for replying but No, I think it is not the best solution for RTL direction 
after ```left: auto;```

![image](https://user-images.githubusercontent.com/19759247/30910011-d48032e2-a38b-11e7-8213-d7d66b5a2688.png)

Reply:
@OmarJaber That problem is for ltr as well. 

![image](https://user-images.githubusercontent.com/9355208/30960071-9038aed2-a45f-11e7-90de-cf802c44bb97.png)